.github/workflows/publish.yml

```diff
@@ -28,11 +28,11 @@ jobs:
         id: changed_files
         with:
           files: |
-             packages/react-magma-docs/src/data/properties.json
+             packages/react-magma-d/src/data/properties.json
```

Comment:
was this an accident? or is this something to catch both `dom` and `docs`?

Reply:
accident 😄